common/formatting/token_partition_tree_test.cc

```diff
@@ -398,4 +398,486 @@ TEST_F(MoveLastLeafIntoPreviousSiblingTest, TwoGenerations) {
                                     << *diff.right << '\n';
 }
 
+class TokenPartitionTreeWrapperTest
```

Comment:
Related to earlier suggestion in token_partition_tree.h.
Is it possible to formulate these `TokenPartitionTreeWrapperTest`s in terms of only `ReshapeFittingSubpartitions`, the public interface?

Reply:
Fixed.